ably/realtime_client_test.go

```diff
@@ -47,6 +49,42 @@ func TestRealtime_RealtimeHost(t *testing.T) {
 	}
 }
 
+func TestReatime_RSC7_AblyAgent(t *testing.T) {
+	t.Parallel()
+
+	app, err := ablytest.NewSandbox(nil)
```

Comment:
Why does this test need a Sandbox app?

Reply:
It needs for generating `auth token`, if not used, it fails internally with implicit auth request

Reply:
Can't it just use a fake key? This puts needless load on our Sandbox cluster.

Reply:
Actually I am not quite sure, how to do it

Reply:
For example:

```go
client, err := ably.NewRealtime(ably.WithKey("xxx:xxx"))
```

Reply:
I will check with existing implementation

Reply:
`WithKey` does't work, because it doesn't suppoer non-TLS, had to use `withToken`, so done